src/pages/index.vue

```diff
@@ -845,6 +847,8 @@ export default {
       dialogNFTClassList: [],
       isSiteHeaderFixed: false,
       searchQuery: '',
+      placeholderText: '',
+      randomKeywords: [],
```

Comment:
can be computed props

Reply:
[♻️ Refactor randomKeywords into a computed prop](https://github.com/likecoin/liker-land/pull/2003/commits/754a2a13427587e7b9fc7b4cddbca97d77348386)